RevenueCatUI/CustomerCenter/Views/ManageSubscription/ManageSubscriptionView.swift

```diff
@@ -177,80 +170,75 @@ struct ManageSubscriptionsView: View {
             $0.navigationTitle(self.viewModel.screen.title)
                 .navigationBarTitleDisplayMode(.inline)
          })
-        .onChangeOf(purchaseInformation?.customerInfoRequestedDate) { _ in
-            guard let purchase = purchaseInformation else { return }
-            viewModel.reloadPurchaseInformation(purchase)
-        }
```

Comment:
this is not happening anymore. Any reason?

Reply:
Just re-added it. Good catch